### DISCUSSION TITLE
Get displayed nickname of user

### DISCUSSION THREAD

**Discussion by TheRisenPhoenix** [discord-jda/JDA#1934] [Questions and Help] [answered]
Members of a server can change their displayed nickname for this server.
However, as far as I can see, a bot can only recieve the real username of a user.
Is there a way to also get the server nickname for this user, or to somehow look it up?

**Comment by RedcodesDev** [comment, marked as answer]
You can use Member#getEffectiveName() to get the Nickname or if it is unset the real username of a guild member. You can read it in the docs here: https://ci.dv8tion.net/job/JDA/javadoc/net/dv8tion/jda/api/entities/Member.html#getEffectiveName()

**Reply by TheRisenPhoenix** [reply]
Thank you very much! :)

**Reply by DV8FromTheWorld** [reply]
In contrast, you can _explicitly_ get the Nickname using https://ci.dv8tion.net/job/JDA/javadoc/net/dv8tion/jda/api/entities/Member.html#getNickname()

If a nickname is not set, this will return `null`
